frontend/src/state/reducers/userPlansReducer.ts

```diff
@@ -224,40 +235,58 @@ export const userPlansReducer = (
           return draft;
         }
 
-        const activePlan = draft.plans[draft.activePlan!];
+        // active plan name should always be definied
+        const activePlanName = draft.activePlan || "";
+
+        const activePlan = draft.plans[activePlanName];
```

Comment:
Is there a scenario when we index `draft.plan` by ""? Curious if that is possible and if we need to worry about index errors.

Reply:
not really actually, what I could do for now is just add an if check just to be sure

Reply:
Naw it's good as it is 👍